packages/share/example/android/app/src/main/java/io/flutter/plugins/shareexample/EmbeddingV1Activity.java

```diff
@@ -1,18 +0,0 @@
-// Copyright 2017 The Chromium Authors. All rights reserved.
-// Use of this source code is governed by a BSD-style license that can be
-// found in the LICENSE file.
-
-package io.flutter.plugins.shareexample;
-
-import android.os.Bundle;
-import io.flutter.app.FlutterActivity;
-import io.flutter.plugins.GeneratedPluginRegistrant;
-
-public class EmbeddingV1Activity extends FlutterActivity {
```

Comment:
Why was this deleted? If it shouldn't be deleted, can we bring it back?

Reply:
Ah true. Guess I deleted too much :) Will bring it back.

Reply:
Noticed now, why I switched totally to v2. Reason is, that using v1 didn't work because path_provider seems to only work with v2? Build works fine with v1 example as it is now in the share example. But as soon as we add path_provider (used in the example to store a text file and share it), it will not work with the error:

MissingPluginException(No implementation found for method getApplicationDocumentsDirectory on channel plugins.flutter.io/path_provider)

Any recommendation on what to do?

Sorry for the force pushes. Trying to clean up the commit log with all the small forth and back.

Reply:
Never mind. Had to add the path provider plugin manually in the MainActivity.java to have it available.